### DISCUSSION TITLE
GitHub Student Developer Pack: Uni Email needed !

### DISCUSSION THREAD

**Discussion by ZainAsif767** [community/community#72745] [GitHub Education]
### Select Topic Area

Bug

### Body

Hello! I've run into an issue with the GitHub Student Developer Pack. Unfortunately, my university doesn't provide school email addresses. I initially used my email for the first four attempts, but my fifth application was rejected with the requirement to use a university-provided email address.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ZainAsif767 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by ZeeshanMukhtar1** [comment]
Hi @ZainAsif767! Unfortunately, the requirement to use a university-provided email address is a strict one, and there's not much that can be done to work around it.

However, I recommend a few things you could try:

- Contact your university's IT department and ask if they can provide you with a school email address. Even if they don't normally offer this service, they may be able to make an exception for you.
- If you have a personal website or blog, you could try using that as your verification method instead of a school email address. GitHub may accept this as an alternative form of verification.
- If all else fails, you could try reaching out to GitHub support directly and explaining your situation. They may be able to offer some guidance or assistance.

I hope this helps! Let me know if you have any other questions.

**Reply by ZainAsif767** [reply]
@ZeeshanMukhtar1 Thanks man, I verified and submitted all the required documents and was successful in the first four verifications, but the 5th attempt was rejected. I couldn't understand why every time I was using my ID card to get verified. I'll try what you have suggested.

**Reply by ZeeshanMukhtar1** [reply]
I look forward to seeing you as a verified Student and best of luck with your future attempt..!
dear **Zain** if please consider my answer "**AS Marked**" Big Thanks to you zain 🥰

**Reply by ZeeshanMukhtar1** [reply]
Please let me know if u have any other tech confusion , I will be happy to guide you 😊
